lib/usd/translators/shading/mtlxBaseReader.cpp

```diff
@@ -152,4 +140,79 @@ bool MtlxUsd_BaseReader::GetColorAndAlphaFromInput(
     return false;
 }
 
+bool MtlxUsd_BaseReader::TraverseUnconnectableInput(const TfToken& usdAttrName)
+{
+    TfToken               usdPortName;
```

Comment:
Considers a constructor node to be part of the set of MaterialX nodes that define a Maya node.